coins/safecash.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "safecash",
+    "symbol": "SCASH",
+    "algorithm": "equihash",
+    "parameters": {
+            "N": 192,
+            "K": 7,
+            "personalization": "SafeCash"
+    },
+    "requireShielding": false,
+    "txfee": 0.0001,
+    "peerMagic": "5a45524f",
+    "payFoundersReward": true,
+    "payAllFounders": true,
+    "foundersRewardAddresses": ["SPfqCriDDxsaH854z3YHigCNUPfSdN3sr1","SaSmUn7LkjmV2WUH1dKFK1xZR1FMRT4qrn","SbfnZN1rPdFq4WsfUUjxphGFcfEits7JXH","SSW62dCjdCxNbKiZdmJh5GLepauY1J1rXU","SW9GN4tZVfPftAQvGmZQLbK6NqePSs8fX1"],
```

Comment:
Can you fix the alignment in this file? We've put an https://editorconfig.com [file](https://github.com/s-nomp/s-nomp/blob/master/.editorconfig) in the repo that should help with styling if your IDE supports it. Primarily, four space indentation, expand arrays and objects (i.e. lines 18-22 not being on one line), etc. Feel free to check out the other configs for ideas.